Jenkinsfile

```diff
@@ -1,117 +1,43 @@
 #!/usr/bin/groovy
 
-// Copyright (C) Pelagicore AB 2017
+// Copyright (C) Luxoft Sweden AB 2018
```

Comment:
Actually, is this correct? Shouldn't the old copyright still be there?

Reply:
This is technically correct since Pelagicore AB changed name to Luxoft Sweden AB. And while changing the name I updated the year since we are now in 2018.